### PR TITLE
billing page

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: ui_src/src/components/datePicker/index.js [hunk 1]
```diff
@@ -19,25 +19,25 @@ const DatePickerComponent = ({ width, height, minWidth, onChange, placeholder, p
...
 
     const disabledDate = (current) => {
-      // Disable months before January 2023
-      const disabledBefore = current && current < new Date('2023-06-01');
-      // Disable months after the current month
-      const disabledAfter = current && current > new Date();
-  
-      return disabledBefore || disabledAfter;
+        // Disable months before January 2023
+        const disabledBefore = current && current < new Date('2023-06-01');
```
Comment (avrhamNeeman) [inline ui_src/src/components/datePicker/index.js:23]: '2023-06-01' should be dynamic

File: ui_src/src/components/datePicker/index.js [hunk 2]
```diff
@@ -19,25 +19,25 @@ const DatePickerComponent = ({ width, height, minWidth, onChange, placeholder, p
     const [disabledMonths, setDisabledMonths] = useState([]);
 
     const disabledDate = (current) => {
-      // Disable months before January 2023
-      const disabledBefore = current && current < new Date('2023-06-01');
-      // Disable months after the current month
-      const disabledAfter = current && current > new Date();
-  
-      return disabledBefore || disabledAfter;
+        // Disable months before January 2023
```
Comment (avrhamNeeman) [inline ui_src/src/components/datePicker/index.js:22]: Please remove comments

File: ui_src/src/components/datePicker/index.js [hunk 3]
```diff
@@ -19,25 +19,25 @@ const DatePickerComponent = ({ width, height, minWidth, onChange, placeholder, p
...
-        const disabledMonths = [];
-        let currentDate = new Date();
-        while (currentDate > new Date('2023-06-01')) {
-          disabledMonths.push(currentDate);
-          currentDate = new Date(currentDate.getFullYear(), currentDate.getMonth() - 1);
+        if (open) {
+            // Generate the list of disabled months
+            const disabledMonths = [];
+            let currentDate = new Date();
+            while (currentDate > new Date('2023-06-01')) {
```
Comment (avrhamNeeman) [inline ui_src/src/components/datePicker/index.js:35]: The same

File: ui_src/src/domain/overview/tags/index.js [hunk 4]
```diff
@@ -27,25 +27,27 @@ const Tags = () => {
...
-                                        <label className="item-num">{`${index + 1}`}</label>
-                                        <Tag tag={{ color: tag.color, name: tag.name }} onClick={() => ''}></Tag>
+                    <div className="tags-items-container">
+                        {state?.monitor_data?.tags_details?.map((tag, index) => {
+                            return (
+                                <div key={index}>
+                                    <span className="tag-item">
+                                        <span className="item">
+                                            <label className="item-num">{`${index + 1}`}</label>
+                                            <Tag tag={{ color: tag.color, name: tag.name }} onClick={() => ''}></Tag>
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/tags/index.js:37]: Don't pass onClick and handle it inside the Tag component.
Also, tags shouldn't be clickable.

File: ui_src/src/domain/overview/tags/index.js [hunk 5]
```diff
@@ -27,25 +27,27 @@ const Tags = () => {
...
+                                        </span>
+                                        <label className="attached-component">
+                                            {' '}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/overview/tags/index.js:40]: Remove

File: ui_src/src/services/valueConvertor.js [hunk 6]
```diff
@@ -383,6 +385,7 @@ export const compareVersions = (a, b) => {
 };
 
 export const isCloud = () => {
-    const cloud_env = process.env.REACT_APP_CLOUD === 'true';
-    return cloud_env;
+    return true;
+    // const cloud_env = process.env.REACT_APP_CLOUD === 'true';
```
Comment (avrhamNeeman) [inline ui_src/src/services/valueConvertor.js:389]: remove comments

File: ui_src/src/components/datePicker/index.js (hunk 1 above)
Comment (SvetaMemphis) [reply ui_src/src/components/datePicker/index.js:23]: Done

File: ui_src/src/components/datePicker/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/components/datePicker/index.js:22]: done

File: ui_src/src/components/datePicker/index.js (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/components/datePicker/index.js:35]: done

File: ui_src/src/domain/overview/tags/index.js (hunk 4 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/overview/tags/index.js:37]: done

File: ui_src/src/services/valueConvertor.js (hunk 6 above)
Comment (SvetaMemphis) [reply ui_src/src/services/valueConvertor.js:389]: done

File: ui_src/src/components/datePicker/index.js [hunk 7]
```diff
@@ -15,29 +15,27 @@ import './style.scss';
...
     const [disabledMonths, setDisabledMonths] = useState([]);
 
     const disabledDate = (current) => {
-      // Disable months before January 2023
-      const disabledBefore = current && current < new Date('2023-06-01');
-      // Disable months after the current month
-      const disabledAfter = current && current > new Date();
-  
-      return disabledBefore || disabledAfter;
+        const startingDate = dateFrom ? new Date(dateFrom) : new Date('2023-06-01');
```
Comment (avrhamNeeman) [inline ui_src/src/components/datePicker/index.js:22]: 2023-06-01 ?

**Pull request merged**